### DISCUSSION TITLE
formatting individual link

### DISCUSSION THREAD

**Discussion by pigwa88** [nfrasser/linkifyjs#421] [Q&A]
Hi, I cant understand how I can format few links in individual way... for example github.com with rel:follow (not working).
How I can building something like a IF: _"if(url=mywebiste.com){rel: follow,}"_ in const options?? is it possible?
(I will add that all formatting data will come from class=post)
```js
const beforeEl = document.getElementById('linkify-demo-before')
const afterEl = document.getElementById('linkify-demo-after')

const options = {
  title: "External Link",
  rel: 'nofollow noreferrer noopener',
  target: '_blank',

  formatHref: {
    hashtag: (val) => `https://www.twitter.com/hashtag/${val.substr(1)}`,
    mention: (val) => `https://github.com/${val.substr(1)}`,
  }
}

linkifyHtml("github.com", {
  rel: 'follow',
 });

afterEl.innerHTML = linkifyStr(beforeEl.value, options)
beforeEl.addEventListener('input', () => {
  afterEl.innerHTML = linkifyStr(beforeEl.value, options)
  
})
```

**Comment by nfrasser** [comment]
Hi @pigwa88, most Linkify options support a function value that accepts the link, link type and link token as arguments and returns the desired option value. For `rel`, you could specify the option like this:

```js
const options = {
  rel: (val, type, token) => {
    if (val.includes('github.com') {
      return 'follow';
    } else {
      return 'nofollow';
    }
  }
};
```

See the `format` option docs for more examples on how to use this: https://linkify.js.org/docs/options.html#format. Note the `rel` option also accepts a function or an object with function values.
